src/test/java/org/llorllale/cactoos/matchers/HasValuesTest.java

```diff
@@ -42,81 +40,74 @@
  * @since 1.0.0
  * @checkstyle MagicNumberCheck (500 lines)
  * @checkstyle JavadocMethodCheck (500 lines)
+ * @checkstyle RegexpSinglelineCheck (500 lines)
  * @checkstyle MultilineJavadocTagsCheck (500 lines)
+ * @checkstyle StringLiteralsConcatenationCheck (500 lines)
  */
 @SuppressWarnings("PMD.AvoidDuplicateLiterals")
 public final class HasValuesTest {
 
     @Test
-    public void contains() {
+    public void matches() {
         MatcherAssert.assertThat(
-            new HasValues<>("a", "b").matchesSafely(
-                new ListOf<>("a", "b", "c", "e"),
-                new StringDescription()
-            ),
-            new IsEqual<>(true)
+            "The matcher check that [1,2,3] contains [2]",
+            new ListOf<>(1, 2, 3),
+            new HasValues<>(2)
         );
     }
 
-    /**
-     * Verify the matcher message in case when test failed.
-     * @todo #32:30m Matcher to test {@link org.hamcrest.Matcher} objects.
-     *  In order to check the matchers we should verify that
-     *  - error message is correct in case false status
-     *  - the status of testing is correct itself.
-     *  The possible name - {@code IsMatcherOf}.
-     */
     @Test
-    public void containsErrorMessage() {
-        final Description msg = new StringDescription();
+    public void contains() {
```

Comment:
@dgroup I think this one can be called `matchSafely`

Reply:
@llorllale done.